### ISSUE TITLE
paypal payment method new order email

### ISSUE THREAD

**Issue by papppeter** [OFFLINE-GmbH/oc-mall-plugin#241]
Hi,

if i place an order and pay it with paypal customer gets email about the success of the payment, but i don't get the new order email. It is working with offline payment provider.

all the notifications are enabled.

**Comment by tobias-kuendig** [comment]
Do you see any errors in your system.log file?

**Comment by papppeter** [comment]
i search for offline.mall but i could not find anything.

**Comment by tobias-kuendig** [comment]
Just to be sure: The customer receives the Mall generated email (not the one from PayPal) but you (the admin) do not receive anything?

**Comment by papppeter** [comment]
yes. that is the case.

but i receive this on offline payment provider.

**Comment by papppeter** [comment]
HI, one more question, would it be possible to add payment info to the emails as well? offline payment i have an information of the bank account they need to transfer the money.

**Comment by tobias-kuendig** [comment]
This is fixed in 7842be6. The off-site payment did not trigger the mall.checkout.succeeded event correctly.

> HI, one more question, would it be possible to add payment info to the emails as well?

If you are using the default partials this is already available:

```twig
{{ order.payment_method.renderInstructions(order)  | raw }}
```

see https://github.com/OFFLINE-GmbH/oc-mall-plugin/blob/develop/views/mail/_partials/order/payment_state.htm#L8

**Comment by papppeter** [comment]
thank you!
